src/spikeinterface/core/base.py

```diff
@@ -798,7 +799,9 @@ def load(file_path: Union[str, Path], base_folder: Optional[Union[Path, str, boo
             return extractor
 
         else:
-            raise ValueError("spikeinterface.Base.load() file_path must be an existing folder or file")
+            error_msg = f"{file_path} is not a file or a folder. It should point out to either a json, pickle file or a
+            folder that is the result of extractor.save(...)"
```

Comment:
```suggestion
            error_msg = f"{file_path} is not a file or a folder. It should point out to either a json, pickle file or a "
            "folder that is the result of extractor.save(...)"
```
the lines were missing quotes :)